### PR TITLE
Use current settings object in steps that require a settings object.

### PR DESCRIPTION
**Pull request by mfoltzgoogle**

### CODE REVIEW

**Comment by mfoltzgoogle** [comment]
Addresses Issue #336: PresentationRequest constructor uses entry settings object, but it probably shouldn't.

File: index.html [hunk 1]
```diff
@@ -1521,9 +1525,9 @@
             <li>If one of the following conditions is true:
               <ul>
                 <li>The result of running the <a>prohibits mixed security
-                contexts algorithm</a> on the document's <a>settings object</a>
-                is <code>"Prohibits Mixed Security Contexts"</code> and any
-                member of <var>presentationUrls</var> is an <a>a priori
+                contexts algorithm</a> on the document's <a>current settings
```
Comment (domenic) [inline index.html]: Documents don't have current settings objects. They have relevant settings objects.

File: index.html [hunk 2]
```diff
@@ -443,9 +443,11 @@
         WHATWG URL standard [[!URL]].
       </p>
       <p>
-        The term  <dfn><a href=
+        The terms <dfn><a href=
         "https://html.spec.whatwg.org/multipage/webappapis.html#concept-current-everything">
-         current settings object</a></dfn> is defined in [[HTML]].
+        current settings object</a></dfn> and <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-everything">relevant
```
Comment (domenic) [inline index.html]: Should go to https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object

File: index.html [hunk 3]
```diff
@@ -443,9 +443,11 @@
         WHATWG URL standard [[!URL]].
       </p>
       <p>
-        The term  <dfn><a href=
+        The terms <dfn><a href=
         "https://html.spec.whatwg.org/multipage/webappapis.html#concept-current-everything">
```
Comment (domenic) [inline index.html]: Should go to https://html.spec.whatwg.org/multipage/webappapis.html#current-settings-object

**Comment by domenic** [comment]
LGTM, thanks so much!

**Pull request merged**